### ISSUE TITLE
The plug-in «Customise toolbar» works no more in N++ 8.4.2 and newer versions

### ISSUE THREAD

**Issue by SN-CH** [notepad-plus-plus/notepad-plus-plus#12461]
It must have some thing to do with the fix [https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9913](https://github.com/notepad-plus-plus/notepad-plus-plus/issues/9913).

N++ 8.4.2 refuses to launch while the file `Toolbaricons.xml` is in folder `C:\Users\<user_name>\AppData\Roaming\Notepad++`. Should I rename that file, the program starts, but the standard set of icons (which I don't like, no offence to any one, it is just that my taste is different) is shown, of course.

It is `Fluent UI: small` what is selected in `Preferences` (other options do not work with the icon set I use).

I tried to follow that discussion in the thread mentioned above, but failed at understanding which specific steps I should take to get the latest versions of N++ to work with the plug-in (which has version 5.3). Any explanation would be much appreciated.

**Comment by alankilborn** [comment]
Your questions rightly belong on the _Customize Toolbar_ support site, but I don't know that there really is one.  Barring that, you really should discuss it on the N++ Community site [HERE](https://community.notepad-plus-plus.org/).

As it is not an "issue" for Notepad++ core, it doesn't belong here.

-------------------------

In regards to the broad statement made in the title:

>The plug-in «Customise toolbar» works no more in N++ 8.4.2 and newer versions

I can say that I'm very successfully running N++ v8.4.6 and Customize Toolbar plugin v5.3.

**Comment by SN-CH** [comment]
Regarding your first comment, I suppose you are right. I'll move the question to the forum you suggested.

Regarding the second one, I am not so sure. At the moment, I am forced to use N++ 8.4.1. As soon as I install 8.4.2 or anything above (I actually had tried various versions, before I found 8.4.1 to be the most recent version that works with the plug-in, that was exactly the reason for the broad statement as you called it), the program stops launching. I tried to remove everything completely, including the profile before posting here. I must admit I do not know how to explain the fact it works for you, though.

The plug-in has not been changed for a while. On the other hand, the fix I have cited above has something to do with the same functionality. So, my guess is the fix might have had some unintended consequences (if so, it is not a plug-in issue).
